src/pages/public_registration/PublicRegistrationContent.tsx

```diff
@@ -110,6 +110,15 @@ export const PublicRegistrationContent = ({ registration }: PublicRegistrationCo
               }}>
               {t('registration.public_page.result_not_published')}
             </Typography>
+            <Typography
```

Comment:
Jeg hadde en annen oppfatning av hva "Add following tekst to blue box" betydde da, og trodde det var boksen i høyremeyen 🤔 Vet JEG har snakka om at vi trenger tekst som beskriver hva publisering/DOI/kuratorstøtte-boksene faktisk er. Men mulig jeg tar feil her altså.
Nå vil denne bare vises for kladder også, men det er like relevant å vise for publiserte resultat også ville jeg tenkt

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/f50b7989-c7ed-46d3-adb1-9f8ad7c09405)